geniza/corpus/sitemaps.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+from django.contrib.sitemaps import Sitemap
+from django.urls import reverse
+
+from geniza.corpus.models import Document
+from geniza.corpus.solr_queryset import DocumentSolrQuerySet
+
+
+def solr_timestamp_to_date(timestamp):
+    """Convert solr isoformat date time string to python date."""
+    # format: 2020-05-12T15:46:20.341Z
+    # django sitemap only includes date, so strip off time
+    yearmonthday = timestamp.split("T")[0]
+    return date(*[int(val) for val in yearmonthday.split("-")])
```

Comment:
parasolr includes a `solr_timestamp_to_datetime` utility method now, let's just use that and then convert to date (I'm guessing it wasn't available yet in whatever code you're adapting from)

Reply:
8cbc9204b6d6cf7f291826b228163520bbf78746 ✅